samples/v1.0/Elements/Image.ImageBaseUrl.json

```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+  "version": "1.0",
+  "backgroundImage": "Assets/background.jpg",
+	"body": [
+		{
+			"type": "Image",
+			"url": "Assets/Adaptive10.png"
+		}
+	],
+  "actions": [
+    {
+      "type": "Action.ShowCard",
```

Comment:
>       [](start = 0, length = 6)

The spacing in this file is inconsistent (some tabs some spaces). Please use tabs in the .json files.